shared-module/displayio/TileGrid.c

```diff
@@ -219,10 +219,19 @@ void common_hal_displayio_tilegrid_set_tile(displayio_tilegrid_t *self, uint16_t
     } else {
         tile_area = &temp_area;
     }
-    tile_area->x1 = x * self->tile_width;
+    int16_t tx = (x - self->top_left_x) % self->width_in_tiles;
+    if (tx < 0) {
+        tx += self->height_in_tiles;
```

Comment:
should this be ``width_in_tiles``?

Reply:
Yup! Terminal only changes y so I didn't notice. Good catch!